### DISCUSSION TITLE
easy way for external squelch open indication?

### DISCUSSION THREAD

**Discussion by 9a4db** [rtl-airband/RTLSDR-Airband#318] [General]
My experience with RTLSDR-Airband is poor but have nice other SDR SW and HW experience.
Looking for easy way to have sort of external indication when certain channel is active / squelch open.
For example channel of local airport active for 5 seconds GREEN LED light, channel of Radar freq active 3 sec RED light.
Can use combination of RPI v2 with arduino uno serial over the serial / usb for actual indications,
just looking how to trigger all that from RTLSDR - Airband? 
Know about level indications and * signaling squelch open in -f mode.

Have here on hand arduino UNO, RTL SDR v2 stick and RPI 2B+ for testing.

Thank you...

**Comment by flux-girdles** [comment]
see #306
